
class DuplicateUrlError(ValueError):
    pass


class UrlNotFound(Exception):
    pass
